src/Model/GServer.php

```diff
@@ -326,198 +343,220 @@ private static function getBaseURL(string $url): string
 	public static function detect(string $url, string $network = '', bool $only_nodeinfo = false): bool
 	{
 		Logger::info('Detect server type', ['server' => $url]);
-		$serverdata = ['detection-method' => self::DETECT_MANUAL];
 
 		$original_url = $url;
 
 		// Remove URL content that is not supposed to exist for a server url
-		$url = self::cleanURL($url);
-
-		// Get base URL
-		$baseurl = self::getBaseURL($url);
+		$url = rtrim(self::cleanURL($url), '/');
```

Comment:
Shouldn't the `rtrim(<url>, '/')` call included in GServer::cleanURL? If not, why do we need it here specifically?

Reply:
We are using that function at a another place as well, that I haven't changed: https://github.com/friendica/friendica/blob/develop/src/Worker/UpdateGServer.php#L55

I wanted to avoid side effects.